### PR TITLE
add new map Volcano

### PR DESCRIPTION
**Pull request by ksandronline**

### CODE REVIEW

**Comment by moreavy** [comment]
@ksandronline you need to choose a license for your map. You can check out https://github.com/MT-CTF/capturetheflag/tree/master/mods/ctf/ctf_map/map_maker#license

**Comment by ksandronline** [comment]
moreavy thanks. im add license. if need do more - im ready.

**Comment by moreavy** [comment]
@ksandronline you could simply choose a license from the CC, GNU or OpenSource

**Comment by winniepee** [comment]
There's some errors which I fixed, here's the folder for the map i changed
[volcano.zip](https://github.com/MT-CTF/maps/files/6625522/volcano.zip)

**Comment by ksandronline** [comment]
@winniepee hello. please check map. What need do more??? need more island? maybe add to game and check in real battle?

**Comment by olliy1or** [comment]
@ksandronline you have extra 2 files that's not needed `volcano.zip` and `volcano/license`, having the license inside the config file is enough

**Review by Panquesito7 — COMMENTED** [review]
The screenshot should have the chat and HUD disabled, IMO.

**Comment by ksandronline** [comment]
Im have question: im add ores, but in game they not don't dig. Name blocks: ctf_map:stone_with_mese, ctf_map:stone_with_diamond, ctf_map:stone_with_iron, ctf_map:stone_with_coal. How to make fossils digable?

**Comment by ksandronline** [comment]
Im find rocket in game. Im can use shooter_rocket:rocket and shooter_rocket:rocket_gun in this map? Im can add to treasures? Im do small test and like destroy with rocket. Maybe one two rocket not would it hurt every team?

**Comment by moreavy** [comment]
> Im find rocket in game. Im can use shooter_rocket:rocket and shooter_rocket:rocket_gun in this map? Im can add to treasures? Im do small test and like destroy with rocket. Maybe one two rocket not would it hurt every team?

I think it was an old class called "Rocketeer", and shooter:_rocket:rocket and shooter:_rocket:rocket may be the class items. I'm not sure because non of the map nor does CTF in crafting have rockets.

**Review by moreavy — APPROVED** [review]
Noice Map!

File: volcano/map.conf [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+name = Volcano
+
+h = 140
+r = 115
+rotation = z
+
+initial_stuff = grenades:smoke,shooter:pistol,default:wood 50,default:ladder_wood 50,default:axe_steel,default:torch 10
+base_node = default:steelblock
+
+# this im copy from tunnel map (i like crossbow)
```
Comment (JostP) [inline volcano/map.conf:13]:
```suggestion
# I copied this from tunnel map (i like crossbow)
```

File: volcano/map.conf [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+treasures = default:ladder,0.3,2,10,20;default:glass,0.3,2,5,15;default:cobble,0.4,5,45,99;default:stone,0.2,3,10,20;default:wood,0.3,5,30,60;default:gravel,0.3,3,10,20;default:obsidian,0.06,2,3,6;default:steel_ingot,0.15,2,5,10;default:mese_crystal,0.08,2,2,4;default:diamond,0.05,2,1,2;doors:door_steel,0.3,5,1,3;ctf_traps:damage_cobble,0.3,4,10,20;ctf_map:reinforced_cobble,0.3,4,10,20;default:sword_mese,0.1,3,1,2;default:sword_diamond,0.06,2,1,2;default:shovel_mese,0.1,3,1,2;default:shovel_diamond,0.06,2,1,2;default:axe_steel,0.15,2,1,2;default:axe_mese,0.1,3,1,2;default:axe_diamond,0.06,2,1,2;default:pick_diamond,0.1,2,1,2;shooter:shotgun,0.05,2,1;grenades:frag,0.1,2,1;grenades:smoke,0.1,2,1;shooter:machine_gun,0,03,2,1;shooter:crossbow,0.2,2,1,2;shooter:pistol,0.4,2,1,4;shooter:ammo,0.3,2,1,10;shooter:arrow_white,0.3,2,10,20;medkits:medkit,0.8,5,2;ctf_bandages:bandage,0.8,2,3,6
```
Comment (JostP) [inline volcano/map.conf:14]: Well, I thought you liked sniper rifles ;)

```suggestion
treasures = default:ladder,0.3,2,10,20;default:glass,0.3,2,5,15;default:cobble,0.4,5,45,99;default:stone,0.2,3,10,20;default:wood,0.3,5,30,60;default:gravel,0.3,3,10,20;default:obsidian,0.06,2,3,6;default:steel_ingot,0.15,2,5,10;default:mese_crystal,0.08,2,2,4;default:diamond,0.05,2,1,2;doors:door_steel,0.3,5,1,3;ctf_traps:damage_cobble,0.3,4,10,20;ctf_map:reinforced_cobble,0.3,4,10,20;default:sword_mese,0.1,3,1,2;default:sword_diamond,0.06,2,1,2;default:shovel_mese,0.1,3,1,2;default:shovel_diamond,0.06,2,1,2;default:axe_steel,0.15,2,1,2;default:axe_mese,0.1,3,1,2;default:axe_diamond,0.06,2,1,2;default:pick_diamond,0.1,2,1,2;shooter:shotgun,0.05,2,1;grenades:frag,0.1,2,1;grenades:smoke,0.1,2,1;shooter:machine_gun,0,03,2,1;shooter:crossbow,0.2,2,1,2;shooter:pistol,0.4,2,1,4;shooter:ammo,0.3,2,1,10;shooter:arrow_white,0.3,2,10,20;medkits:medkit,0.8,5,2;sniper_rifles:rifle_762_loaded,0.15,2,1;sniper_rifles:rifle_magnum_loaded,0.04,2,1
```

File: volcano/map.conf [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+
+h = 140
+r = 115
+rotation = z
+
+initial_stuff = grenades:smoke,shooter:pistol,default:wood 50,default:ladder_wood 50,default:axe_steel,default:torch 10
+base_node = default:steelblock
+
+# I copied this from tunnel map (i like crossbow)
+treasures = default:ladder,0.3,2,10,20;default:glass,0.3,2,5,15;default:cobble,0.4,5,45,99;default:stone,0.2,3,10,20;default:wood,0.3,5,30,60;default:gravel,0.3,3,10,20;default:obsidian,0.06,2,3,6;default:steel_ingot,0.15,2,5,10;default:mese_crystal,0.08,2,2,4;default:diamond,0.05,2,1,2;doors:door_steel,0.3,5,1,3;ctf_traps:damage_cobble,0.3,4,10,20;ctf_map:reinforced_cobble,0.3,4,10,20;default:sword_mese,0.1,3,1,2;default:sword_diamond,0.06,2,1,2;default:shovel_mese,0.1,3,1,2;default:shovel_diamond,0.06,2,1,2;default:axe_steel,0.15,2,1,2;default:axe_mese,0.1,3,1,2;default:axe_diamond,0.06,2,1,2;default:pick_diamond,0.1,2,1,2;shooter:shotgun,0.05,2,1;grenades:frag,0.1,2,1;grenades:smoke,0.1,2,1;shooter:machine_gun,0,03,2,1;shooter:crossbow,0.2,2,1,2;shooter:pistol,0.4,2,1,4;shooter:ammo,0.3,2,1,10;shooter:arrow_white,0.3,2,10,20;medkits:medkit,0.8,5,2;sniper_rifles:rifle_762_loaded,0.15,2,1;sniper_rifles:rifle_magnum_loaded,0.04,2,1
```
Comment (JostP) [inline volcano/map.conf:14]:
```suggestion
treasures = default:ladder,0.3,2,10,20;default:glass,0.3,2,5,15;default:cobble,0.4,5,45,99;default:stone,0.2,3,10,20;default:wood,0.3,5,30,60;default:gravel,0.3,3,10,20;default:obsidian,0.06,2,3,6;default:steel_ingot,0.15,2,5,10;default:mese_crystal,0.08,2,2,4;default:diamond,0.05,2,1,2;doors:door_steel,0.3,5,1,3;ctf_traps:damage_cobble,0.3,4,10,20;ctf_map:reinforced_cobble,0.3,4,10,20;default:sword_mese,0.1,3,1,2;default:sword_diamond,0.06,2,1,2;default:shovel_mese,0.1,3,1,2;default:shovel_diamond,0.06,2,1,2;default:axe_steel,0.15,2,1,2;default:axe_mese,0.1,3,1,2;default:axe_diamond,0.06,2,1,2;default:pick_diamond,0.1,2,1,2;shooter:shotgun,0.05,2,1;grenades:frag,0.1,2,1;grenades:smoke,0.1,2,1;shooter:machine_gun,0,03,2,1;shooter:crossbow,0.2,2,1,2;shooter:pistol,0.4,2,1,4;shooter:ammo,0.3,2,1,10;shooter:arrow_white,0.3,2,10,20;medkits:medkit,0.8,5,2;sniper_rifles:rifle_762_loaded,0.15,2,1;sniper_rifles:rifle_magnum_loaded,0.04,2,1;default:pick_steel,0.3,2,1,10
```

File: volcano/map.conf [hunk 4]
```diff
@@ -0,0 +1,34 @@
...
+
+author = ksandr
+name = Volcano
+
+h = 140
+r = 115
+rotation = z
+
+initial_stuff = grenades:smoke,shooter:pistol,default:wood 50,default:ladder_wood 50,default:axe_steel,default:torch 10
+base_node = default:steelblock
```
Comment (JostP) [inline volcano/map.conf:11]:
```suggestion
initial_stuff = grenades:smoke,shooter:pistol,default:ladder_wood 50,default:pick_stone,default:torch 10
base_node = ctf_map:steelblock
```

File: volcano/map.conf [hunk 5]
```diff
@@ -0,0 +1,34 @@
+license = CC BY-SA 4.0
+
+author = ksandr
+name = Volcano
+
+h = 140
+r = 115
+rotation = z
+
+initial_stuff = shooter:pistol,grenades:smoke,default:ladder_wood 50,default:pick_stone,default:torch 10
```
Comment (JostP) [inline volcano/map.conf:10]:
```suggestion
rotation = z

hint = Look for a treasure hidden inside the vulcano!
initial_stuff = shooter:pistol,grenades:smoke,default:ladder_wood 50,default:pick_stone,default:torch 10
```

**Comment by JostP** [comment]
@ksandronline I made some small changes to your map. Check them and tell me if you approve them. Map can be added after your approval.

Nice map :+1:

**Comment by RyanZem** [comment]
> @ksandronline I made some small changes to your map. Check them and tell me if you approve them. Map can be added after your approval.
> 
> Nice map

I won't be able to play when this will be new :crying_cat_face:

File: volcano/map.conf (hunk 5 above)
Comment (DiamondPlane) [reply volcano/map.conf]: "vulcano" or "volcano"?

File: volcano/map.conf [hunk 6]
```diff
@@ -0,0 +1,35 @@
+license = CC BY-SA 4.0
+
+author = ksandr
+name = Volcano
+
+h = 140
+r = 115
+rotation = z
+
+hint = Look for a treasure hidden inside the vulcano!
```
Comment (DiamondPlane) [inline volcano/map.conf:10]: Shouldn't it be "volcano" instead of "vulcano"?

Comment (JostP) [reply volcano/map.conf:10]: oh yes, my bad

Comment (JostP) [reply volcano/map.conf:10]: thx

Comment (JostP) [reply volcano/map.conf:10]:
```suggestion
hint = Look for a treasure hidden inside the volcano!
```

**Comment by ksandronline** [comment]
JostP im approve. Me need press special button in interface github? (its me first request on github.)
And me interest: Im can use shooter_rocket:rocket and shooter_rocket:rocket_gun ? or shooter_rocket:rocket_gun status WIP? thanks.

**Pull request merged**